extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -644,39 +662,38 @@ export class AmpPanZoom extends AMP.BaseElement {
   }
 
   /**
-   * Performs a one-step pinch zoom action.
-   * @param {number} centerClientX
-   * @param {number} centerClientY
-   * @param {number} deltaX
-   * @param {number} deltaY
-   *  @param {number} dir
-   * @private
+   * @param {number} clientX
+   * @param {number} clientY
    */
-  onPinchZoom_(centerClientX, centerClientY, deltaX, deltaY, dir) {
-    this.zoomToPoint_(centerClientX, centerClientY, deltaX, deltaY, dir);
+  onDoubletapZoom_(clientX, clientY) {
+    const newScale = this.scale_ == 1 ? this.maxScale_ : this.minScale_;
```

Comment:
What happens when `this.minScale_`  is not 1? Seems like this should be:

```javascript
const newScale = this.scale_ == this.minScale_ ? this.maxScale_ : this.minScale_;
```

Reply:
Good catch. You're right.

Reply:
Though technically per current implementation minScale will always be 1.